README.md

```diff
@@ -24,9 +24,31 @@ This makes your app feel super fast, regardless of server location or internet c
 | uses immutable.js behind the scenes                    | uses native JS objects behind the scenes                          |
 | FSA compliant                                          | not FSA compliant                                                 |
 | must wrap your state calls in `ensureState`            | no change necessary to get your state                             |
-##Usage
 
-###Feed it your reducer
+## How do optimistic actions work?
+
+- When you dispatch a `BEGIN` action
+  - Your current state is recorded as the `beforeState`.
```

Comment:
not necessarily, this is true for the first BEGIN, but then for the 2nd BEGIN, the beforeState doesn't change